cql3/statements/create_view_statement.cc

```diff
@@ -212,6 +212,12 @@ future<shared_ptr<cql_transport::event::schema_change>> create_view_statement::a
         if (dynamic_pointer_cast<selection::selectable::writetime_or_ttl::raw>(selectable)) {
             throw exceptions::invalid_request_exception(format("Cannot use function when defining a materialized view"));
         }
+        if (auto e = dynamic_pointer_cast<selection::selectable::with_expression::raw>(selectable)) {
```

Comment:
Looks like this change got into a wrong patch? It doesn't seem related to making column_identifier::raw forward declarable.

Reply:
Yes, it belongs to the patch introducing with_expression.

Reply:
Fixed. Also moved the definition of with_expression::raw::expression().